client/components/spotlight/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+
+const SpotlightContainer = styled.div`
+	background-color: var( --studio-gray-0 );
```

Comment:
```suggestion
	background-color: var( --studio-gray-0 );
	border-radius: 5px;
```
To match plugin card radius. Much like spacing we should work towards using borderRadii values like `var(--radius-2)`

Reply:
Can we use these now?

Reply:
@lsl yes we can!